pkg/envoy/cds/builder.go

```diff
@@ -28,9 +31,139 @@ import (
 	"github.com/openservicemesh/osm/pkg/trafficpolicy"
 )
 
+type clusterBuilder struct {
+	proxyIdentity identity.ServiceIdentity
+
+	outboundMeshTrafficPolicy *trafficpolicy.OutboundMeshTrafficPolicy
+
+	inboundMeshTrafficPolicy *trafficpolicy.InboundMeshTrafficPolicy
+
+	sidecarSpec configv1alpha2.SidecarSpec
+
+	egressTrafficPolicy *trafficpolicy.EgressTrafficPolicy
+
+	envoyOutboundPassthroughCluster string
```

Comment:
nit: I would consider leaving this as a non-field, and continue to use the constant value in its place